### DISCUSSION TITLE
Is it possible to update an existing whisper?

### DISCUSSION THREAD

**Discussion by hans-hsieh-asus** [open-olive/loop-development-kit#140] [Q&A]
Let's say I have several checkboxes in a whisper for different options, and the first checkbox's label is "select all". The behavior I want is when the user checks the first checkbox, all the other checkboxes will become checked. Is it possible the update a whisper that is already created?

**Comment by cshields-oliveai** [comment]
@hans-hsieh-asus   - It is not currently possible to update an existing Whisper. This work is scheduled in the Olive Helps roadmap however, and we expect to pull this work in soon. We will send out updates in release notes when it is included in the LDK release.

In order to offer the highest level support and categorize issues appropriately, we have decided that the "Issues" tab on Github will be our main method of communication. As consequence of this, we are going to be turning off the "Discussions" capability for this repository. Please keep bringing us good points over there!

If you would like to get a notification when Whisper update capability has been added, please open a Github Issue and add an "enhancement" label.
